### ISSUE TITLE
Missing converter: triple-equals

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#187]
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 1.2.0
-   `typescript-eslint` version: 2.3.2

#### TSLint Rule Name

`triple-equals`

**Comment by JoshuaKGoldberg** [comment]
The issue here is that this package doesn't know how to convert the TSLint rule into the equivalent ESLint rule. Accepting PRs to add in a converter! 🙌 

[`docs/Development.md`](https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/Development.md) has a root overview for how to get started. Within that, [`docs/Architecture.md#rule-converters`](https://github.com/typescript-eslint/tslint-to-eslint-config/blob/master/docs/Architecture.md#rule-converters) talks about how these converters operate. Check the [typescript-eslint roadmap](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md) to see suggestions on equivalent ESLint rules to convert to.

See #189 for an example of a simple rule converter that was recently added in.

**Comment by Mouzourides** [comment]
Hey I'd like to give this a go

**Comment by Mouzourides** [comment]
The TSLint options for the triple-equals rule are: [ "allow-null-check", "allow-undefined-check" ].
The ESLint options for eqeqeq rule are: ["smart", "always"]

The "allow-null-check" option best maps to the ESLint "smart "option. It also has the additional functionality of allowing comparing two literal values and evaluating the value of typeof. I have added this information in the "notice" array if the user has selected the "allow-null-check" option.
There is a way to only ignore null comparisons by selecting the ESLint "always" option with additional configuration e.g. `["error", "always", {"null": "ignore"}]` however I believe it's encouraged to use the "smart" option and so I have implemented this.

The "allow-undefined-check" option does not have a direct mapping to ESLint. I have added this to the "notice" array and selected the "smart" option as I believe it to be the closest.

Not supplying an argument for TSLint defaults to the ESLint "always" option.
